### PR TITLE
Re-enable Codechef is_invalid_handle_check

### PR DESCRIPTION
**Pull request by punndcoder28**

### CODE REVIEW

File: modules/sites/codechef.py [hunk 1]
```diff
@@ -199,29 +199,9 @@ def rating_graph_data(handle):
...
-        json_data = response.json()
-        if json_data["result"]["data"]["code"] == 9003:
-            return NOT_FOUND
-        else:
-            return "VALID_HANDLE"
-
+    
     # --------------------------------------------------------------------------
+    @staticmethod
     def __get_access_token(self):
```
Comment (raj454raj) [inline modules/sites/codechef.py]: staticmethod won't require self argument right?

File: modules/sites/codechef.py [hunk 2]
```diff
@@ -252,9 +232,31 @@ def __get_access_token(self):
                                      ex=TTL_TIME)
             return access_token
         else:
-            print "Error requesting CodeChef API for access token"
+            print("Error requesting CodeChef API for access token")
```
Comment (raj454raj) [inline modules/sites/codechef.py]: keep it as it is, this script is executed with python 2

File: modules/sites/codechef.py [hunk 3]
```diff
@@ -252,9 +232,31 @@ def __get_access_token(self):
...
             return
 
+    # --------------------------------------------------------------------------
+    @staticmethod
+    def __validate_handle(self):
```
Comment (raj454raj) [inline modules/sites/codechef.py]: no self

File: modules/sites/codechef.py [hunk 4]
```diff
@@ -252,9 +232,31 @@ def __get_access_token(self):
...
+        """
+            Make an API request to Codechef to see if a user exists
+        """
+
+        response = get_request("%s/users/%s" % (CODECHEF_API_URL, self.handle),
```
Comment (raj454raj) [inline modules/sites/codechef.py]: You will have to take handle as an argument as self won't be available.

File: modules/sites/codechef.py [hunk 5]
```diff
@@ -252,9 +232,31 @@ def __get_access_token(self):
...
+                               headers={"User-Agent": user_agent,
+                                        "Authorization": "Bearer %s" % self.access_token},
```
Comment (raj454raj) [inline modules/sites/codechef.py]: you will have to take access_token as an argument

**Comment by raj454raj** [comment]
Hey @punndcoder28 , still working on this?

**Comment by punndcoder28** [comment]
Hey @punndcoder28. I was trying to install all the requirements and get the project up and running but while doing so I encountered a lot of errors and couldn't figure out why.

**Comment by raj454raj** [comment]
Closing due to inactivity. @punndcoder28 please re-open after testing.

**Pull request closed without merging**